src/components/PersonLinks/PersonLinks.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Person } from '../../types';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+export const PersonLinks: React.FC<Props> = ({ person, people }) => {
+  const { slug: id } = useParams();
+
+  const findedPersonByName = (name: string | null) => {
```

Comment:
There is a typo in the function name `findedPersonByName`. It should be `findPersonByName` to follow standard English grammar and naming conventions.